cmd/netstate-grpc/server/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/grpclog"
+
+	pb "github.com/storj/protos/netstate/netstate"
+)
+
+var (
+	port int
+)
+
+func initializeFlags() {
+	flag.IntVar(&port, "port", 8080, "port")
+	flag.Parse()
+}
+
+func main() {
+	err := Main()
+	if err != nil {
+		log.Fatalf("fatal error: %v", err)
+		os.Exit(1)
+	}
+}
+
+func Main() {
+	initializeFlags()
+
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", port))
+	if err != nil {
+		grpclog.Fatalf("failed to listen: %v", err)
+	}
+	var opts []grpc.ServerOption
+	grpcServer := grpc.NewServer(opts...)
+	pb.RegisterNetStateServer(grpcServer, newServer())
+	grpcServer.Serve(lis)
```

Comment:
You should listen for a close and shutdown gracefully

Reply:
I'm going to delete/re-do this file but will note for the future!